Trainer_v5/Trainer.Source/TrainerBehaviour.cs

```diff
@@ -1061,10 +1061,14 @@ public static void TakeoverCompanyAction(string input)
 
 			var simulatedCompanyWorth = simulatedCompany.GetPossibleStockWorth();
 
-			simulatedCompany.BuyOut(new Company[]
-			{
-				Settings.MyCompany
-			}, false);
+			simulatedCompany.BuyOut(
+				new Company[] { Settings.MyCompany }, // companies buying out
+				false,                                // not broke
+				SDateTime.Now(),                      // current time
+				true                                  // can disconnect (default value)
+			);
+
+			Settings.MyCompany.MakeTransaction(simulatedCompanyWorth, Company.TransactionCategory.Stocks, null, false);
```

Comment:
@muddxyii You added a new Settings.MyCompany.MakeTransaction, but we already have it below so it would make two transactions.

Reply:
You're absolutely correct, I must've skipped looking right over that. I removed the duplicate transaction.